### ISSUE TITLE
PBR texture 

### ISSUE THREAD

**Issue by iekiekkeikei** [mateuskreuch/minecraft-miniature-shader#9]
I like PBR textures.
Is there any chance that PBR textures will be supported by this shader?

**Comment by iekiekkeikei** [comment]
×PBR 
❍labPBR

**Comment by NolanHewitt** [comment]
Integrated PBR like complementary has would be cool but might be out of scope for this shader

**Comment by mateuskreuch** [comment]
Yeah, PBR textures are probably not going to happen, at least by me. I personally don't use them, and the way lighting is handled by Miniature is very unconventional which makes this harder than it seems.

**Comment by iekiekkeikei** [comment]
I see. That's a bit disappointing. 
I like the shader and look forward to seeing how it evolves in the future.
